src/components/nodes/boundaryEvent/boundaryEvent.vue

```diff
@@ -147,7 +146,7 @@ export default {
     },
     resetToInitialPosition() {
       this.shape.position(this.validPosition.x, this.validPosition.y);
-      this.allowSetNodePosition = true;
+      store.commit('allowSavingState');
```

Comment:
Could we add a method to the store that encapsulates this? As it stands this doesn't really clarify what's going on well.